.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
 # Learn how to add code owners here:
 # https://help.github.com/en/articles/about-code-owners
 
-*                 @jaredpalmer @gsoltis @gaspar09 @nathanhammond @tknickman @chris-olszewski @mehulkar
+* @Turbo-OSS
```

not sure why GH is flagging this. @jaredpalmer does this team have Write access to the repo?

---

You need to scope this and lowercase it. 

```suggestion
* @vercel/turbo-oss 
```